src/com/nutrons/framework/util/FlowOperators.java

```diff
@@ -25,4 +27,18 @@
   public static <T> Flowable<T> toFlow(Supplier<T> supplier) {
     return toFlow(supplier, 100, TimeUnit.MILLISECONDS);
   }
+
+  /**
+   * Modifies motor input to prevent motors running at very small values
+   *
+   * @param input the raw motor input values
+   * @return the motor input values with values smaller than 0.2 removed
+   */
+  public static Flowable<Double> deadband(Flowable<Double> input) {
```

Comment:
Add second function `deadbandMap` which is the lambda so you can do the transformation as a map instead of a wrapping function call.